src/core/i18n/utils.js

```diff
@@ -246,7 +246,7 @@ export function makeI18n(
   i18nData: I18nConfig,
   lang: string,
   _Jed: Jed = Jed,
-  { _Intl = Intl }: makeI18nOptions = {}
+  { _Intl = window.Intl }: makeI18nOptions = {}
```

Comment:
This is [used in server code](https://github.com/mozilla/addons-frontend/blob/master/src/core/server/base.js#L32) so it needs to be protected by `if (typeof window !== 'undefined')...` or something like that.